src/index.js

```diff
@@ -486,17 +469,11 @@ class ImageMinimizerPlugin {
         compilation.hooks.assetPath.tap(
           { name: pluginName },
           (filename, data, info) => {
-            const newInfo =
-              data &&
-              // @ts-ignore
-              data.module &&
-              // @ts-ignore
-              data.module.buildMeta &&
-              // @ts-ignore
-              data.module.buildMeta.imageMinimizerPluginInfo;
+            // @ts-ignore
+            const newInfo = data?.module?.buildMeta?.imageMinimizerPluginInfo;
 
-            if (newInfo) {
-              Object.assign(info || {}, newInfo);
+            if (info && newInfo) {
```

Comment:
Before when we don't have `info` we always write it, now we write newInfo only when `info` have a value

Reply:
Imagine we have code:
```js
let newInfo = { x: 42 };
let info = undefined;
Object.assign(info || {}, newInfo);
```
this has no any affect, because you write properties into fresh created object (`{}`), but does not store it to any variable

Reply:
Oh, I see, missed full logic (sometimes it is hard to review due bad github diff and you don't see all logic, I think we are using `info` in webpack API, but here we only assign)